src/domUpdates.js

```diff
@@ -13,8 +14,81 @@ const domUpdates = {
 
   populateDestinationDropDown: (destination) => {
     $('#select-destination').append(`<option value="${destination.destination}">${destination.destination}</option>`)
-  }
+  },
 
+  //If cannot be tested yet. Need to do a POST
+  populateTravelerInfo: (traveler, travelerID, tripData, destinationData) => {
+    if (traveler.findCurrentTrips(travelerID, tripData)) {
+      let tripDestination = destinationData.find(destination => traveler.findCurrentTrips(travelerID, tripData).destinationID === destination.id)
+      $('.display-username').text(`Welcome ${traveler.name}, enjoy ${tripDestination.destination}!`)
+    } else {
+      $('.display-username').text(`Welcome ${traveler.name}!`)
+    }
+    $('.display-username').text(`Welcome ${traveler.name}!`)
```

Comment:
I think line 27 will always override the text from the conditional (line 23) so it seems like you have an extra/repeated line

Reply:
OH! I think youre right